tsl/test/shared/expected/transparent_decompress_chunk-15.out

```diff
@@ -49,14 +54,15 @@ FROM :TEST_TABLE t
 WHERE device_id IN (1, 2)
 ORDER BY time, device_id;
 QUERY PLAN
- Sort (actual rows=7196 loops=1)
-   Sort Key: t."time", t.device_id
-   Sort Method: quicksort 
-   ->  Result (actual rows=7196 loops=1)
+ Result (actual rows=7196 loops=1)
+   ->  Sort (actual rows=7196 loops=1)
+         Sort Key: t."time", t.device_id
+         Sort Method: quicksort 
```

Comment:
This looks like a regression, why does it no longer use index scan?

Reply:
Couple other places look suspicious as well

Reply:
Yeah I don't like these plans as well, and the only thing I did is added ANALYZE. Maybe the table must be made bigger... It seems that the compressed table is very small in some queries (e.g. about 20 rows), so it starts to prefer seq scan.

Reply:
The scan below decompress_chunk shouldnt be affected statistics changes as those are not picked based on cost

Reply:
Looking at the `create_compressed_scan_path`, it does make cost-based decisions on seq scan vs index scan (it adds them using the normal `add_path`), so I guess this might be the reason for what we're seeing here.